doc/release-notes/4.20-release-notes.md

```diff
@@ -126,7 +128,7 @@ We made changes to the JSON Export in this release (Issue 6650, PR #6669). If yo
 ## New JVM Options for file storage drivers
 
 - The JVM option dataverse.files.file.directory=<your directory> controls where temporary files are stored (in the /temp subdir of the defined directory), independent of the location of any 'file' store defined above.
-- The JVM option dataverse.files.<id>.upload-redirect enables direct upload of files added to a dataset to the S3 bucket. (S3 stores only!)
+- The JVM option dataverse.files.<id>.upload-redirect enables direct upload of files added to a dataset in the S3 bucket. (S3 stores only!)
```

Comment:
Why are you editing a release note for a very old release?

Reply:
This was by accident (I did search for all occurrences of the relevant jvm option and found this file and realized to late it was old release notes). I have reverted it.